payment-request/onpaymentmenthodchange-attribute.https.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<!-- Copyright © 2017 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
```

Comment:
I've noticed you remove this comment before and it's not included on the other two tests.